doc/whats_new/v0.24.rst

```diff
@@ -406,13 +406,17 @@ Changelog
   :user:`Joseph Willard <josephwillard>`
 
 - |Fix| bug in :func:`metrics.hinge_loss` where error occurs when
-  ``y_true`` is missing some labels that are provided explictly in the
+  ``y_true`` is missing some labels that are provided explicitly in the
   ``labels`` parameter.
   :pr:`17935` by :user:`Cary Goltermann <Ultramann>`.
 
 :mod:`sklearn.model_selection`
 ..............................
 
+- |MajorFeature| :class:`model_selection.GroupTimeSeriesSplit` is a Time Series
```

Comment:
this needs to move to the next release (doc/whats_new/v1.0.rst)

Reply:
Done